src/relative-time-element.js

```diff
@@ -4,6 +4,21 @@ import RelativeTime from './relative-time'
 import ExtendedTimeElement from './extended-time-element'
 import {localeFromElement} from './utils'
 
+// Internal: Array tracking all elements attached to the document that need
+// to be updated every minute.
+const nowElements = new Set()
+
+// Internal: Timer ID for `updateNowElements` interval.
+let updateNowElementsId
+
+// Internal: Install a timer to refresh all attached relative-time elements every
+// minute.
+function updateNowElements() {
+  clearTimeout(updateNowElementsId)
```

Comment:
shouldn't this reset the updateNowElementsId variable too ? Otherwise, we could end up in a case where the timer won't run anymore:

1. add an element in the DOM. The `connectedCallback` will run the update and schedule a timer
2. remove the element from the DOM.
3. wait a bit for the timer to run.  `nowElements.size` is 0 so the timer is not rescheduled. but currently, `updateNowElementsId` keeps the old integer for the timer that just ran
4. add another element in the DOM. The `connectedCallback` will **not** run the update, because `updateNowElementsId` is already set (except that this is not set for a scheduled timer but for an old one). so the timer is never restarted